src/fidesops/service/storage/storage_uploader_service.py

```diff
@@ -111,15 +105,4 @@ def _local_uploader(
 ) -> str:
     """Uploads data to local storage, used for quick-start/demo purposes"""
     file_key: str = _construct_file_key(request_id, config)
-    if not os.path.exists(LOCAL_FIDES_UPLOAD_DIRECTORY):
-        os.makedirs(LOCAL_FIDES_UPLOAD_DIRECTORY)
-    with open(f"{LOCAL_FIDES_UPLOAD_DIRECTORY}/{file_key}", "w") as f:
-        json.dump(data, f, default=_handle_json_encoding)
-    return "success"
-
-
-def _handle_json_encoding(field: Any) -> str:
-    """Specify str format for datetime objects"""
-    if isinstance(field, datetime):
-        return field.strftime("%Y-%m-%dT%H:%M:%S")
-    return field
+    return upload_to_local(data, file_key, request_id)
```

Comment:
This was just moved for circular import reasons when I was adding encryption to local storage -